lib/media/preference_based_criteria.js

```diff
@@ -17,51 +17,33 @@ goog.require('shaka.util.LanguageUtils');
 /**
  * @implements {shaka.media.AdaptationSetCriteria}
  * @final
+ * @export
```

Comment:
Technically I think it is not necessary to export it because it is the default solution.

Reply:
ah, yeah, let me double-check if it's still needed. Added it for the same reason as example-based-criteria.